.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release the component
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main" ]
+    paths:
+      - 'CHANGELOG.md'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    container: codetent/crypta-dev:1.0
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Add env vars
+      run: |
+        echo GORELEASER_CURRENT_TAG=$(changie latest) >> $GITHUB_ENV
+        echo RELEASE_NOTES_PATH=.changes/$(changie latest).md >> $GITHUB_ENV
+    - name: Create release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
not sure if this is required or already implicitly set?

Reply:
I guess this is needed